### PR TITLE
Clean up some subsystem things

### PR DESCRIPTION
**Pull request by Arokha**

### CODE REVIEW

**Comment by Novacat** [comment]
Changes affecting Polaris files should be sent to Polaris.

**Comment by Arokha** [comment]
Already made PR to them for it

File: code/modules/vore/eating/bellymodes_vr.dm [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
 
 	if(loc != owner)
 		if(istype(owner))
 			loc = owner
 		else
 			qdel(src)
-			return SSBELLIES_PROCESSED
-
-	next_process = times_fired + (6 SECONDS / wait) //Set up our next process time.
+			return
```
Comment (Leshana) [inline code/modules/vore/eating/bellymodes_vr.dm]:
```suggestion
			return PROCESS_KILL
```
It just deleted itself, so removing from list is good?  Although... I guess Destroy() will handle it.

Comment (Arokha) [reply code/modules/vore/eating/bellymodes_vr.dm]: Hm! I suppose. Though it will do the work twice, yes. No point since STOP in Destroy().

**Pull request merged**